projects/js-toolkit/packages/liferay-cli/src/new/facet-configuration/index.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * SPDX-FileCopyrightText: © 2017 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: LGPL-3.0-or-later
+ */
+
+import {
+	FilePath,
+	TRANSFORM_OPERATIONS,
+	TemplateRenderer,
+	format,
+	transformTextFile,
+} from '@liferay/js-toolkit-core';
+import fs from 'fs';
+
+import prompt from '../util/prompt';
+
+import type {Facet, Options} from '../index';
+
+const {
+	Text: {appendLines},
+} = TRANSFORM_OPERATIONS;
+const {info, print} = format;
+
+const facet: Facet = {
+	async prompt(useDefaults: boolean, options: Options): Promise<Options> {
+		return await prompt(useDefaults, options, [
```

Comment:
So I am using the tool locally but I don't ever see this prompt, is there a certain path I need to take to see this?

Reply:
That prompt doesn't appear because of this [commit](https://github.com/liferay/liferay-frontend-projects/pull/584/commits/5b816fa23d748d16d28580bdf1372289ff3fe1fb).

The code is here because it has been ported from the old generator and we will need it in the future, for other target types. But I don't think we should give the option to disable configuration or localization for these new projects, since they are specifically targeting Liferay Portal.

We offer the possibility to opt out in other projects in case people don't want to couple their projects with our APIs, but in this case that doesn't apply. Also, even in the other cases, removing the configuration or localization is a matter of deleting the generated files, so not a big deal.

Makes sense?